packages/cozy-jobs-cli/README.md

```diff
@@ -20,6 +20,7 @@ You can use `COZY_URL` to connect to another cozy.
 ### `cozy-konnector-dev`
 
 - Provides `COZY_CREDENTIALS` to the locally running connector from `konnector-dev-config.json` (which is created if it does not exist)
+- A token will be generated (`.token.json`) the first time to grant access to `cozy-konnector-dev`, and you may need to delete it and regenerate it again after a while.
```

Comment:
What is a `while`? Do you mean that the token can be expired? 

While you are at it, I'm wondering if you shouldn't add also that if you change the manifest, you should delete the file too (with a link to this issue) https://github.com/konnectors/libs/issues/701

Reply:
Yes token can be expired.

Oh yes good point for the manifest

Reply:
> it again after a while. 

This sentence is not precise enough to be part of the documentation. You should talk about expired token instead.